src/types/plugin-input.ts

```diff
@@ -0,0 +1,28 @@
+import { SupportedEvents, SupportedEventsU } from "./context";
+import { StaticDecode, Type as T } from "@sinclair/typebox";
+
+export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU extends SupportedEvents[T] = SupportedEvents[T]> {
+  stateId: string;
+  eventName: T;
+  eventPayload: TU["payload"];
+  settings: StartStopSettings;
+  authToken: string;
+  ref: string;
+}
+
+export const startStopSchema = T.Object({
+  disabledCommands: T.Array(T.String()),
```

Comment:
Should have defined values for options (I guess "start", "stop")

Reply:
I think just "start" makes sense.

It allows a user to stop a task if the command should be disabled but does not allow a new hunter to start, what do you think?

Reply:
It does make more sense but I don't think this would be a very common scenario. So in the end I would say let's have both there or none.

Reply:
understood, I've added stop as default as well